src/main/java/io/github/thebusybiscuit/slimefun4/implementation/resources/OrganicNetherGoo.java

```diff
@@ -0,0 +1,35 @@
+package io.github.thebusybiscuit.slimefun4.implementation.resources;
+import org.bukkit.inventory.ItemStack;
+
+import io.github.thebusybiscuit.slimefun4.api.events.PlayerRightClickEvent;
+import io.github.thebusybiscuit.slimefun4.api.items.ItemSetting;
+import io.github.thebusybiscuit.slimefun4.core.attributes.NotPlaceable;
+import io.github.thebusybiscuit.slimefun4.core.attributes.PiglinBarterDrop;
+import io.github.thebusybiscuit.slimefun4.core.handlers.ItemUseHandler;
+import io.github.thebusybiscuit.slimefun4.implementation.items.SimpleSlimefunItem;
+import me.mrCookieSlime.Slimefun.Lists.RecipeType;
+import me.mrCookieSlime.Slimefun.Objects.Category;
+import me.mrCookieSlime.Slimefun.api.SlimefunItemStack;
+
+public class OrganicNetherGoo extends SimpleSlimefunItem<ItemUseHandler> implements NotPlaceable, PiglinBarterDrop {
```

Comment:
Why is this even marked as `NotPlaceable` here? Brown dye cannot be placed anyway.